src/main/java/org/broadinstitute/hellbender/utils/GenotypeCounts.java

```diff
@@ -2,25 +2,25 @@
 
 public final class GenotypeCounts {
 
-    private final int ref;
-    private final int het;
-    private final int hom;
+    private final double ref;
```

Comment:
Add comment here that these are `doubles` rather than `ints` in order to match GATK3 behavior, in case someone else gets outraged over this and attempts the same refactor Adam did.